manuscript_heb/00-Introduction.md

```diff
@@ -2,171 +2,155 @@
 
-<B> פרק 2 : Strings ו Regular Expressions </B> הפרק סוקר פונקציונאליות נוספת חדשה לעיבוד ומניפולאציות על מחרוזות string ו בחינה  וכן מציג הקונספט החדש  template strings. 
+כל אחד מ-13 הפרקים בספר מכסה אספקט אחר ב-ECMAScript 6. פרקים רבים מתחילים בדיון על הבעיות ש-ECMAScript 6 נועדה לפתור, על מנת להציג יריעה רחבה לסיבות לשינויים אלו.
+כל פרק מכיל מספר דוגמאות קוד על מנת לסייע לך ללמוד מבנה חדש בשפה וקונספטים תכנותיים חדשים.
 
- 
-<B> פרק 3 : Functions in ECMAScript 6 </B>  הפרק עוסק בשינויים לגבי פונקציות.   זה כולל  תבנית פונקציות חץ,  ערך ברירת מחדל לפרמטרים של פונקציה  ונושאים נוספים. 
-  
-<B>   פרק 4 :  הרחבת פונקציונאליות של אוביקטים   </B>  מסביר את השינוי כיצד אוביקטים נוצרים, עוברים שינוי וכיצד משתמשים בהם.  הנושאים כוללים שינוי בליטרל של אוביקט , ומטודות הצגה. 
+<B>פרק 1 : כיצד עובד Block Binding </B> מדבר על `let` ו-`const` - החלפת block-level ב-var.
 
-<B>  פרק 5 : פעולת הרס אוביקטים  עבור גישה פשוטה יותר לנתונים </B>  מציג הרס אוביקטים ומערכים , המאפשר לבמע דקומפוז  של אוביקטים ומערכים בצורת כתיבה קצרה יותר. 
+<B> פרק 2 : Strings ו Regular Expressions </B> סוקר יכולת נוספת חדשה לעיבוד על מחרוזות וכן מציג את הקונספט החדש של template strings.
 
-<B>  פרק 6 :  Symbols ו - Symbols property  </B>  מציג את הקונספט של Symbols, דרך חדשה להגדיר תכונות אוביקט. Symbols הוא סוג חדש של  primitive type שניתן להשתמש בו  בצורה מוסתרת (אבל לא נסתרת לחלוטין )  לגבי תכונות   ומטודות  של  אוביקט . 
+<B> פרק 3 : Functions in ECMAScript 6 </B> עוסק בשינויים הקשורים בפונקציות, שינויים הכוללים את המבנה של פונקציות חץ, ערך ברירת מחדל לפרמטרים של פונקציה ונושאים נוספים.
+ 
+<B> פרק 4 : הרחבת פונקציונאליות של אובייקטים </B> מסביר את השינוי כיצד אובייקטים נוצרים, עוברים שינוי וכיצד משתמשים בהם. הנושאים כוללים שינוי בליטרל של אובייקט, ומתודות הצגה.
 
-<B>  פרק 7 : Sets ו Maps   </B>  מפרט את הסוגים החדשים  של   `Set`, `WeakSet`, `Map`, and `WeakMap`.  סוגים חדשים אלו מרחיבים את השימושיות של מערך  , מבחינת סמנטיקה ת עיבוד ת ניהול זיכרון שהותאם במיוחד ל  JavaScript. 
- 
-<B>  פרק 8 : Iterators ו Generators  </B>  עוסק בתוספת של   Iterators ו Generators  לשפה. , תכונות אלו מאפשרות לך לעבוד עם אוספים של דאטה  בצורה יעילה ועוצמתית  שלא התאפשרה בגרסה הקודמת של  JavaScript. 
+<B> פרק 5 : פעולת הרס אובייקטים עבור גישה פשוטה יותר לנתונים </B> מציג הרס אובייקטים ומערכים, המאפשר לבצע דקומפוז של אובייקטים ומערכים בצורת קצרה ונוחה יותר.
 
-<B> פרק 9 : הצגת   Classes ב JavaScript  </B>  מציג  הקונספט הפורמאלי הראשון  של  Classes ב JavaScript. לעיתים נקודת בילבול בין מה שמקיים בשפות  תכנות אחרות , התוספת של סינטאקס  של  Class ב JavaScript מאפשר שימוש בשפה בצורה יותר  נוחה ובצורה  מקוצרת  למי שמעוניין. 
- 
-<B> פרק 10 : יכולות משופרות של Array  </B>  מפרט השינויים  למערכים  רגילים ודרכים מענינות חדשות בהם מערכים  ניתנים לשימוש ב JavaScript.  
+<B> פרק 6 : Symbols ו - Symbols property </B> מציג את הקונספט של Symbols, דרך חדשה להגדיר תכונות אובייקט. Symbols הוא סוג חדש של primitive type שניתן להשתמש בו בצורה מוסתרת (אבל לא נסתרת לחלוטין) לגבי תכונות ומתודות של אובייקט.
```

Comment:
What about primitive type? should we translate or keep it as is?

Reply:
בינתיים תרגמתי את זה 
״ערך פשוט״